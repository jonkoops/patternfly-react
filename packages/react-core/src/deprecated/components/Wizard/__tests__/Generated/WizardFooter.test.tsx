/**
 * @vitest-environment jsdom
 */
import { render } from '@testing-library/react';
import React from 'react';
import { expect, it } from 'vitest';

import { WizardFooter } from '../../WizardFooter';

it('WizardFooter should match snapshot (auto-generated)', () => {
  const { asFragment } = render(<WizardFooter children={'any'} />);
  expect(asFragment()).toMatchSnapshot();
});
