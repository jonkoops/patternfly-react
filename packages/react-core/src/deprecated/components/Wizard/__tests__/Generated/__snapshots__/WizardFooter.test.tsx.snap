// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`WizardFooter should match snapshot (auto-generated) 1`] = `
<DocumentFragment>
  <footer
    class="pf-v5-c-wizard__footer"
  >
    any
  </footer>
</DocumentFragment>
`;
